Dump/run_filter_scripts.py

```diff
@@ -26,9 +26,6 @@
 #
 from Ska.Shell import getenv, bash
 ascdsenv = getenv('source /home/ascds/.ascrc -r release; source /home/mta/bin/reset_param', shell='tcsh')
-ascdsenv['IPCL_DIR'] = "/home/ascds/DS.release/config/tp_template/P011/"
```

Comment:
Could you confirm that all three paths are set by skare?

Reply:
Sourcing /home/ascds/.ascrc correctly applies the latest values. 
```
[waaron@scrapper-16:38:Shell]$ echo $IPCL_DIR 
IPCL_DIR: Undefined variable.
[waaron@scrapper-16:38:Shell]$ echo $ACORN_GUI 
ACORN_GUI: Undefined variable.
[waaron@scrapper-16:38:Shell]$ echo $LD_LIBRARY_PATH
LD_LIBRARY_PATH: Undefined variable.
[waaron@scrapper-16:38:Shell]$ source /home/ascds/.ascrc
**************************************
*** Welcome to the CXC Data System ***
**************************************
CIAO configuration is complete... 
10.12.2  Tuesday, September 19, 2023
  bindir      : /home/ascds/DS.release/bin
  CALDB       : 4.10.8
setting production environment for Mission Planning...
setting production environment for Proposal applications....
CXC Data System, version: 10.12.2  Tuesday, September 19, 2023
[waaron@scrapper-16:39:Shell]$ echo $IPCL_DIR
/home/ascds/DS.release/config/tp_template/P016/
[waaron@scrapper-16:39:Shell]$ echo $ACORN_GUI
/home/ascds/DS.release/config/mta/acorn/scripts/
[waaron@scrapper-16:39:Shell]$ echo $LD_LIBRARY_PATH
/home/ascds/DS.release/lib:/home/ascds/DS.release/ots/lib:/soft/SYBASE16.0/DataAccess64/ODBC/lib:/soft/SYBASE16.0/DataAccess/ODBC/lib:/soft/SYBASE16.0/OCS-16_0/lib:/soft/SYBASE16.0/OCS-16_0/lib3p64:/soft/SYBASE16.0/OCS-16_0/lib3p:/home/ascds/DS.release/otslib:/opt/X11R6/lib
```
If you go to the `/data/mta/TEST/waaron/acorn_FMT6_failure` test directory. I have both the old script version of $IPCL_DIR and the latest version generated by `.ascrc` in two separate tests, showcasing that the latest version yields FMT6.

Also just to be clear for the sake of future devs looking back on these notes, these variables aren't currently being defined by skare. The convention this script set uses is to define the variables with the `/home/ascds/.ascrc` file. The specific MTA ska3-shiny package they use to define and use these variables is `/data/mta/Script/Python3.8/envs/ska3-shiny/lib/python3.8/site-packages/Ska/Shell/shell.py`. The `getenv` function defines a python dictionary of variables, the following three lines in the script would overwrite those variables. Then whenever the script wanted to run acorn using these environment variables, it would used the `bash` function of the `Ska.Shell` package like so: `bash(cmd, env=ascdsenv)`. The second argument also for passing a python dictionary to set the environment variables and then it exectues the `cmd` string in a bash shell.

Reply:
OK